### PR TITLE
update the fullstack job related scripts

### PR DESCRIPTION
**Pull request by Rozzii**
This PR:

- changes the glean branch to a newer branch that contains new glean features for centos like link aggregation and vlan support

- corrects the name of the variable that holds the address of the BMO repository (BMO_REPO was an incorrect name)

### CODE REVIEW

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
/approve

**Comment by kashifest** [comment]
lgtm

File: ci/scripts/image_scripts/build_ipa.sh [hunk 1]
```diff
@@ -209,7 +209,7 @@ CERT_MANAGER_VERSION="${CERT_MANAGER_VERSION}"
 EOF
     pushd "${BMOPATH}"
     cat << EOF >> "${METADATA_PATH}"
-BMO_REPO="${BMOREPO}"
+BMOREPO="${BMOREPO}"
```
Comment (tuminoid) [inline ci/scripts/image_scripts/build_ipa.sh:212]: This does nothing now

**Review by tuminoid — COMMENTED** [review]
Not following what this is achieving?

/hold

File: ci/scripts/image_scripts/build_ipa.sh (hunk 1 above)
Comment (Rozzii) [reply ci/scripts/image_scripts/build_ipa.sh:212]: this is for the metadata generation, it does save the values to a file.

File: ci/scripts/image_scripts/build_ipa.sh [hunk 2]
```diff
@@ -209,7 +209,7 @@ CERT_MANAGER_VERSION="${CERT_MANAGER_VERSION}"
...
 BMO_BRANCH="${BMOBRANCH}"
```
[the remark this answers is not in the document]
Comment (tuminoid) [reply ci/scripts/image_scripts/build_ipa.sh:213]: Oh yes, I missed the cat into file there.

**Review by tuminoid — APPROVED** [review]
/lgtm
/unhold

**Review by lentzi90 — APPROVED** [review]

**Pull request merged**